doc/source/whatsnew/v1.0.0.rst

```diff
@@ -869,6 +869,7 @@ Groupby/resample/rolling
 - Bug in :meth:`DataFrame.groupby` when using axis=1 and having a single level columns index (:issue:`30208`)
 - Bug in :meth:`DataFrame.groupby` when using nunique on axis=1 (:issue:`30253`)
 - Bug in :meth:`GroupBy.quantile` with multiple list-like q value and integer column names (:issue:`30289`)
+- Bug in :meth:`GroupBy.pct_change` and :meth:`SeriesGroupBy.pct_change` causes ``TypeError`` when ``fill_method`` is ``None`` (GH30463)
```

Comment:
can you use the issue format (see other notes), e.g. ``:issue:``, otherwise they won't render

Reply:
That was such a stupid mistake. Sorry for the carelessness.
Fixed now.